packages/next/next-server/server/font-utils.ts

```diff
@@ -39,8 +43,16 @@ export async function getFontDefinitionFromNetwork(
    * The order of IE -> Chrome is important, other wise chrome starts loading woff1.
    * CSS cascading 🤷‍♂️.
    */
-  result += await getFontForUA(url, IE_UA)
-  result += await getFontForUA(url, CHROME_UA)
+  try {
+    result += await getFontForUA(url, IE_UA)
+    result += await getFontForUA(url, CHROME_UA)
+  } catch (e) {
+    console.warn(
```

Comment:
@ijjk Is this the right way to output a warning message ?

Reply:
We have a log util in `build/output` that can be used here, example usage here https://github.com/vercel/next.js/blob/bfd36bd227695cf93c91477e480b6b2d1e6a6f06/packages/next/cli/next-dev.ts#L74

Reply:
Done! Please check